themes/theme-ios11/pages/Cart/components/Item/components/Product/components/Layout/components/Title/style.js

```diff
@@ -9,25 +9,29 @@ const title = css({
 }).toString();
 
 const menuContainer = css({
+  marginTop: `-${variables.gap.bigger}px`,
   marginRight: `-${variables.gap.big}px`,
-  marginLeft: variables.gap.big,
-  pointerEvents: 'none',
 }).toString();
```

Comment:
Same here than in the other file. Please remove all occurrences of `.toString()`. It is not needed and not wanted anymore. Thanks

Reply:
If i omit the `.toString()` the class is not added to the element, since the `classnames` module which i added to the `ContextMenu` component does not perform the necessary steps. So i'd like to keep the `.toString()` in favour of the `classnames` module.